### PR TITLE
Feature/proposals list

### PR DESCRIPTION
**Pull request by SergKukh**
- added endpoint to jobApi for fetching proposals of job
- implement logic to proposals list page
- added available time to properties-slice and useProperties hook
- added information sticker as separate component
- added PageWrapper component

image

### CODE REVIEW

File: apps/freelance/src/pages/ProposalsList/index.tsx [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
+  StyledProjectName,
+  Wrapper,
+} from './styles';
+import useProposalsList from './useProposalsList';
 
 const ProposalsList = () => {
-  const { skills } = useProperties();
+  const { data, isLoading, isSuccess } = useProposalsList();
+
+  if (isLoading) {
```
Comment (lyudad) [inline apps/freelance/src/pages/ProposalsList/index.tsx]: better make wrapper, as you on each page need spinner when data is loading and have same if|else in each component quite annoying

Comment (SergKukh) [reply apps/freelance/src/pages/ProposalsList/index.tsx]: @lyudad created PageWrapper component with optional props isLoading & isSuccess

File: libs/components/src/lib/proposal-card/index.tsx [hunk 2]
```diff
@@ -24,16 +25,14 @@ import {
 import useProposalCard from './useProposalCard';
 
 interface ProposalCardProps {
-  user: User;
-  skills: Property[];
+  user: User | undefined;
   hourlyRate: number;
-  availableTime: string;
+  availableTime: string | undefined;
```
Comment (lyudad) [inline libs/components/src/lib/proposal-card/index.tsx]: `availableTime?: string`

Comment (SergKukh) [reply libs/components/src/lib/proposal-card/index.tsx]: @lyudad fixed

**Pull request merged**